src/components/dev-hub/project-sign-up-form.js

```diff
@@ -49,10 +81,10 @@ const Form = React.memo(() => {
                 onChange={e => setEmail(e.target.value)}
             />
             <TextArea
-                value={project}
+                value={projectDescription}
                 required
                 placeholder="Project Description"
-                onChange={e => setProject(e.target.value)}
+                onChange={e => setProjectDescription(e.target.value)}
             />
             <SubmitContainer>
                 <Button type="submit" primary>
```

Comment:
Should we disable this onsubmit to prevent multiple POSTs

Reply:
yes!